platform/util/platform-util/pom.xml

```diff
@@ -70,6 +70,12 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.16.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
You could move the version up to platform's `pom` as a property to ensure they stay on the same version (if we want them to). I'm not sure we'd want to move that dependency up but I'm not sure about the pros/cons of that.